views/admin/users.erb

```diff
@@ -139,6 +152,67 @@
 <% append_script_declaration("WDN.initializePlugin('datepickers');") %>
 
 <% append_script_declaration(%q[require(['jquery'], function($) {
+
+    function buildSearchFilterList() {
+         let search = [];
+         // pull the values that the search filters had when the page loaded
+         // prevents a bug where a user could change the search filters without pressing the search button and switch to a page that would no longer be valid
```

Comment:
Remove this comment about switching pages because it does not apply here since pagination was removed.

Reply:
done